code/modules/mob/living/simple_animal/friendly/cat.dm

```diff
@@ -296,3 +296,135 @@
 	if(L.a_intent == INTENT_HARM && L.reagents && !stat)
 		L.reagents.add_reagent("nutriment", 0.4)
 		L.reagents.add_reagent("vitamin", 0.4)
+
+//Cat made
+/mob/living/simple_animal/pet/cat/custom_cat
+	name = "White cat" //Incase it somehow gets spawned without an ID
+	desc = "A cute white catto!"
+	icon_state = "custom_cat"
+	icon_living = "custom_cat"
+	icon_dead = "custom_cat_dead"
+	gender = FEMALE
+	gold_core_spawnable = NO_SPAWN
+	health = 50 //So people can't instakill it
+	maxHealth = 50
+	speak = list("Meowrowr!", "Mew!", "Miauen!")
+	speak_emote = list("wigglepurrs", "mewls")
+	emote_hear = list("meows.", "mews.")
+	emote_see = list("looks at you eagerly for pets!", "wiggles enthusiastically.")
+	gold_core_spawnable = NO_SPAWN
+
+//secretcatchemcode, shh!! Of couse I hide it amongst cats. Also, yes, I expect you, Mr.Maintaner to read and review this, dispite it being hidden and not mentioned in the changelogs.
+//I'm not trying to be sneaky, I'm trying to keep it a secret!
+//I don't know how to do hidden chems like Aurora
+//ChemReactionVars:
+/datum/chemical_reaction/fermi/secretcatchem //DONE
+	name = "secretcatchem"
+	id = "secretcatchem"
+	results = list("secretcatchem" = 0.5)
+	required_reagents = list("stable_plasma" = 0.1, "sugar" = 0.1, "cream" = 0.1, "blood" = 0.1, "slimejelly" = 0.1)
+	required_catalysts = list("felinidmutationtoxin" = 1)
+	required_temp = 600
+	mix_message = "the reaction gives off a meow!"
+	mix_sound = "modular_citadel/sound/voice/merowr.ogg"
+	//FermiChem vars:
+	OptimalTempMin 		= 650
+	OptimalpHMin 		= 0
+	ReactpHLim 			= 2
+	CurveSharpT 		= 0
+	CurveSharppH 		= 0
+	ThermicConstant		= 0
+	HIonRelease 		= 0
+	RateUpLim 			= 0.1
+	FermiChem 			= TRUE
+	FermiExplode 		= FALSE
+	PurityMin 			= 0.2
+
+/datum/chemical_reaction/fermi/secretcatchem/New()
+	//rand doesn't seem to work with n^-e
+	OptimalTempMin 		+= rand(-100, 100)
+	OptimalTempMax 		= (OptimalTempMin+rand(0, 200))
+	ExplodeTemp 		= (OptimalTempMax+rand(0, 200))
+	OptimalpHMin 		+= rand(1, 10)
+	OptimalpHMax 		= (OptimalpHMin + rand(1, 5))
+	ReactpHLim 			+= rand(-2, 2)
+	CurveSharpT 		+= (rand(1, 500)/100)
+	CurveSharppH 		+= (rand(1, 500)/100)
+	ThermicConstant		+= rand(-50, 50)
+	HIonRelease 		+= (rand(-25, 25)/100)
+	RateUpLim 			+= (rand(1, 1000)/100)
+	PurityMin 			+= (rand(-1, 1)/10)
+	var/additions = list("aluminum", "silver", "gold", "plasma", "silicon", "bluespace", "uranium", "milk")

```

Comment:
Travis' run failed, it reports an unknown "aluminum" reagent id.
```suggestion
	var/additions = list("aluminium", "silver", "gold", "plasma", "silicon", "bluespace", "uranium", "milk")
```
Another reason why typepaths would be way better than text strings, and there is currently a draft on tgstation's repo about it.

Nevertheless, read the travis runs' reports next time, they are a step above the average "it compiles".